pom.xml

```diff
@@ -69,6 +70,8 @@
     <opentelemetry-semconv.version>1.25.0-alpha</opentelemetry-semconv.version>
     <!-- from grpc-protobuf -->
     <protobuf.version>3.25.1</protobuf.version>
+    <!-- Netty -->
+    <netty.version>4.1.108.Final</netty.version>
 
     <assertj.version>3.25.3</assertj.version>
```

Comment:
Could we please bump Otel to `1.41.0` (released recently)? thank you

Reply:
Updated in https://github.com/openzipkin-contrib/zipkin-otel/pull/9/commits/bc1598a6b86e8db28fcf05c8fe716c2ed7687902